src/site/content/en/handbook/handbook.11tydata.js

```diff
@@ -50,6 +49,13 @@ module.exports = {
           'handbook/self-assessment-components',
         ],
       },
+      {
+        title: 'Content infrastructure',
+        pathItems: [
+          'handbook/resolving-glitch-issues',
+          'handbook/redirects',
+        ],
+      },
```

Comment:
@samthor or @robdodson need a review of this JS change

Reply:
Actually, I know I approved, but could you make this change? It basically wants you to put them on one line:

```js
        pathItems: ['handbook/resolving-glitch-issues', 'handbook/redirects],
```

Reply:
Yup yup already doing it